### PR TITLE
DC/OS 1.12.5 Release Notes

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

https://jira.mesosphere.com/browse/DCOS-62143

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+# Release summary
+
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment. 
+
+# Issues fixed in DC/OS 1.12.5
+The issues that have been fixed in DC/OS 1.12.5 are listed below. Most change descriptions include one or more issue tracking identifiers for reference.
+
+- The DC/OS diagnostics bundle now includes Mesos state logs content, located at `/var/log/mesos-state.tar.gz`. (DCOS-56403)
+- Added 3 new environment files to Mesos systemd. (DCOS-49092)
```
Comment (vinodkone) [inline pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md]: I would rephrase this as "Added support for user managed environment files to mesos systemd units"

File: pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+- Under heavy load, direct requests to the IAM could be slowed down, making the IAM slow to respond to all requests. Mesos master now uses a login endpoint that goes through Admin Router, reducing the effect of any slow senders. This addresses a rare error condition in which the IAM system can become unavailable, potentially leading to Mesos task launch errors in the strict security mode. (DCOS-56053)
+- Package specific icons are now showing up properly for all services installed from Universe. (DCOS-53674, COPS-4710) 
+- Fixed a licensing permissions error, that resulted in the following message, "error getting LicenseID. Got error: Get http://unix/licenses: dial unix /run/dcos/dcos-licensing.sock: connect: permission denied" (DCOS-57291)
+- Updated code to unify frequency between DC/OS Signal EE and OSS; signal now sends telemetry data every five minutes. (DCOS-57281)
+- Updated DC/OS master to accept the most up to date plugins. Pinned the dcos CLI plugins in every DC/OS branch for both OSS and Enterprise versions, and consolidated all dcos cli instantiations into one fixture. (DCOS-57266)
+- Completed merge signal work for both Enterprise and OSS versions. (DCOS-57062)
```
Comment (vinodkone) [inline pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md]: "Completed work to unify OSS and Enterprise versions of DC/OS Signal."

File: pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+- The DC/OS diagnostics bundle now includes Mesos state logs content, located at `/var/log/mesos-state.tar.gz`. (DCOS-56403)
+- Added 3 new environment files to Mesos systemd. (DCOS-49092)
+- [Metronome] post-install configuration can be added to `/var/lib/dcos/metronome/environment` (DCOS_OSS-5509)
+- Changed `dcos-zk backup` and `dcos-zk restore` to exit early if ZooKeeper is running. (DCOS_OSS-5353)
+- Fixed preflight docker version check failing on unknown versions. (DCOS-56831)
+- DC/OS Net: Fix support for big sets in the IPset manager. (DCOS-57506, COPS-5229)
+- Added new diagnostic REST API bundle with performance improvements. (DCOS_OSS-5098)
+- Fixed old diagnostics bundle from increasing diagnostics job duration when job was done. (DCOS_OSS-5494)
+- Removed `nogroup` creation. (COPS-5220, DCOS-59247)
```
Comment (vinodkone) [inline pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md]: incorrect JIRA: DCOS-59427

Rephrased: "Updated DC/OS installation to not create `nobody` Linux user group on the hosts."

Comment (cmastr) [reply pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md]: Done

File: pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md (hunk 2 above)
Comment (cmastr) [reply pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md]: done

File: pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+- Updated code to unify frequency between DC/OS Signal EE and OSS; signal now sends telemetry data every five minutes. (DCOS-57281)
+- Updated DC/OS master to accept the most up to date plugins. Pinned the dcos CLI plugins in every DC/OS branch for both OSS and Enterprise versions, and consolidated all dcos cli instantiations into one fixture. (DCOS-57266)
+- Completed work to unify OSS and Enterprise versions of DC/OS Signal. (DCOS-57062)
+- Fixed dcos:mesos:master:framework:role:* ACL giving unauthorized access to tasks. (COPS-4929)
+- Fixed job syntax naming bug on Metronome. (COPS-5208)
+- Fixed MacOS Catalina certificate issue with Chrome 77.0.3865.90 for both DC/OS and Konvoy. (COPS-5417)
+- Added correct icons for services that are installed from Universe, but are not classified as Frameworks. (COPS-4727)
+- The time it takes to deploy a service with an L4-VIP has been reduced by pruning VIPs, which reduced growth of state messages exchanged among dcos-net processes. (COPS-5081)
```
Comment (cneth) [inline pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md]: This is a dup of line 51

File: pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+- Package specific icons are now showing up properly for all services installed from Universe. (DCOS-53674, COPS-4710) 
+- Fixed a licensing permissions error, that resulted in the following message, "error getting LicenseID. Got error: Get http://unix/licenses: dial unix /run/dcos/dcos-licensing.sock: connect: permission denied" (DCOS-57291)
+- Updated code to unify frequency between DC/OS Signal EE and OSS; signal now sends telemetry data every five minutes. (DCOS-57281)
+- Updated DC/OS master to accept the most up to date plugins. Pinned the dcos CLI plugins in every DC/OS branch for both OSS and Enterprise versions, and consolidated all dcos cli instantiations into one fixture. (DCOS-57266)
+- Completed work to unify OSS and Enterprise versions of DC/OS Signal. (DCOS-57062)
+- Fixed dcos:mesos:master:framework:role:* ACL giving unauthorized access to tasks. (COPS-4929)
+- Fixed job syntax naming bug on Metronome. (COPS-5208)
+- Fixed MacOS Catalina certificate issue with Chrome 77.0.3865.90 for both DC/OS and Konvoy. (COPS-5417)
```
Comment (cneth) [inline pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md]: It seems weird to me to mention Konvoy in the DC/OS release notes.

File: pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+- Under heavy load, direct requests to the IAM could be slowed down, making the IAM slow to respond to all requests. Mesos master now uses a login endpoint that goes through Admin Router, reducing the effect of any slow senders. This addresses a rare error condition in which the IAM system can become unavailable, potentially leading to Mesos task launch errors in the strict security mode. (DCOS-56053)
+- Package specific icons are now showing up properly for all services installed from Universe. (DCOS-53674, COPS-4710) 
+- Fixed a licensing permissions error, that resulted in the following message, "error getting LicenseID. Got error: Get http://unix/licenses: dial unix /run/dcos/dcos-licensing.sock: connect: permission denied" (DCOS-57291)
+- Updated code to unify frequency between DC/OS Signal EE and OSS; signal now sends telemetry data every five minutes. (DCOS-57281)
+- Updated DC/OS master to accept the most up to date plugins. Pinned the dcos CLI plugins in every DC/OS branch for both OSS and Enterprise versions, and consolidated all dcos cli instantiations into one fixture. (DCOS-57266)
+- Completed work to unify OSS and Enterprise versions of DC/OS Signal. (DCOS-57062)
```
Comment (cneth) [inline pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md]: Does this have some user benefit?  Otherwise it just seems like housekeeping on our end.
